### PR TITLE
increase timeout due to sometimes slow initialization of lmdb, #23284

### PR DESCRIPTION
**Pull request by patriknw**
Refs #23284

### CODE REVIEW

File: akka-distributed-data/src/multi-jvm/scala/akka/cluster/ddata/DurableDataSpec.scala [hunk 1]
```diff
@@ -82,7 +82,7 @@ abstract class DurableDataSpec(multiNodeConfig: DurableDataSpecConfig)
   implicit val cluster = Cluster(system)
 
   val timeout = 5.seconds.dilated
-  val writeTwo = WriteTo(2, timeout)
+  val writeTwo = WriteTo(14, timeout) // initialization of lmdb can be very slow in CI environment
```
Comment (raboof) [inline akka-distributed-data/src/multi-jvm/scala/akka/cluster/ddata/DurableDataSpec.scala]: AFAICS this increases the number of replica's written to, not the timeout - is that what you intended?

**Review by raboof — COMMENTED** [review]
Not sure I understand this test/change?

**Comment by patriknw** [comment]
Looks like I'm in vacation mode already. I'll change

**Review by raboof — APPROVED** [review]

**Review by ktoso — APPROVED** [review]
LGTM :)

**Pull request merged**